tools/validators/instance_validator/tests/entity_instance_test.py

```diff
@@ -288,5 +290,37 @@ def testValidateLinks(self):
     if not instance.IsValidEntityInstance():
       self.fail('exception incorrectly raised')
 
+  def testGoodConnections(self):
```

Comment:
this test could be stricter:  To ensure that this is actually testing what it says it is, the input should be checked to confirm that the entity instance has connections in it and that the connections universe is not null

Reply:
Done